resources-generator/src/main/kotlin/dev/icerock/gradle/AndroidPluginLogic.kt

```diff
@@ -72,9 +73,13 @@ internal class AndroidPluginLogic(
     ): MRGenerator.SourceSet {
         return object : MRGenerator.SourceSet {
             override val name: String
-                get() = "android${androidSourceSet.name.capitalize()}"
+                get() = "android${androidSourceSet.name.replaceFirstChar { 
+                    if (it.isLowerCase()) it.titlecase(Locale.ROOT) else it.toString() }
+                }"
```

Comment:
simplify please

Reply:
reworked

Reply:
create own `capitalize()` extension and use here. simplify code please

Reply:
fixed